a2s/__init__.py

```diff
@@ -1,5 +1,4 @@
-from a2s.exceptions import BrokenMessageError, BufferExhaustedError
-
-from a2s.info import info, ainfo, SourceInfo, GoldSrcInfo
-from a2s.players import players, aplayers, Player
-from a2s.rules import rules, arules
+from a2s.exceptions import BrokenMessageError as BrokenMessageError, BufferExhaustedError as BufferExhaustedError
+from a2s.info import GoldSrcInfo as GoldSrcInfo, SourceInfo as SourceInfo, ainfo as ainfo, info as info
+from a2s.players import Player as Player, aplayers as aplayers, players as players
+from a2s.rules import arules as arules, rules as rules
```

Comment:
What's the reason for changing the imports in this way?

Reply:
This is for "strict" type checking as this PR controls. [PEP561]() and [Pyright's documentation](https://github.com/microsoft/pyright/blob/main/docs/typed-libraries.md#library-interface) explain this much better than I could!

Specifically it'd be this part here:-
> Imported symbols are considered private by default. If they use the “import A as A” (a redundant module alias), “from X import A as A” (a redundant symbol alias), or “from . import A” forms, symbol “A” is not private unless the name begins with an underscore. If a file __init__.py uses the form “from .A import X”, symbol “A” is not private unless the name begins with an underscore (but “X” is still private). If a wildcard import (of the form “from X import *”) is used, all symbols referenced by the wildcard are not private.

Reply:
Pyright mentions the option of using `__all__` to mark imports as public. I'd much prefer that over the redundant aliases.